packages/batch-submitter/src/batch-submitter/batch-submitter.ts

```diff
@@ -17,12 +18,21 @@ export interface ResubmissionConfig {
   gasRetryIncrement: number
 }
 
+interface BatchSubmitterMetrics {
+  batchSubmitterETHBalance: Gauge<string>
+  batchSizeInBytes: Histogram<string>
+  numTxPerBatch: Histogram<string>
+  submissionTimestamp: Histogram<string>
+  submissionGasUsed: Histogram<string>
+}
+
 export abstract class BatchSubmitter {
   protected rollupInfo: RollupInfo
   protected chainContract: Contract
   protected l2ChainId: number
   protected syncing: boolean
   protected lastBatchSubmissionTimestamp: number = 0
+  protected bsMetrics: BatchSubmitterMetrics
```

Comment:
Could just use the name `metrics`; prefix seems unnecessary within this classes scope